themes/vpge/vpge.react_behaviors.yml

```
@@ -0,0 +1,16 @@
+# Example react_behaviors plugin definitions.
+# Plugin property $defaults are defined in ReactBehaviorsPluginManager.
+banner_variant_info:  # part of the key after the :
+  id: banner_type
+  label: 'Banner Paragraphs Type'
+  bundles:
+    - paragraphs_type|stanford_banner
+  config:
+    banner_style:
+      type: select
+      title: 'Banner style'
+      description: Select the variant for displaying this banner.
+#      default_value: vpge-wrapper--banner-default
+      options:
+        vpge-wrapper--banner-default: Default variant
```

Instead of adding the default as an option, it's better to use `empty_option`
```suggestion
      empty_option: Default
      options:
```